docs/API.md

```diff
@@ -212,6 +212,10 @@ The user's username.
 
 The user's session, as returned by the Yggdrasil system. (only client-side)
 
+### client._session
+
+The user's chat session. Only in 1.19.3 and above. You should not need to access this.
```

Comment:
Underscore shouldn't be documented in stable API

Reply:
Are there any unstable API docs in NMP?

Reply:
The API.md is the stable API

Reply:
Sure. But do we have anywhere to document unstable API currently? Or should we just not document this?

Reply:
Yeah you can remove this